SmartCar/Arduino/sketch_main/sketch_main.ino

```diff
@@ -341,6 +357,20 @@ void readFlame(){
       }  
 }
 
+//collision control system  - getting range from all ultrasonic sensors, and printing range of middle one to serial for display
+void collisionControl() {
```

Comment:
Don't ping sonars[0] twice, the value might change and each call to ping_cm() introduces a 20 microseconds delay.